server/src/main/java/org/apache/iotdb/db/metadata/PartialPath.java

```diff
@@ -44,6 +44,10 @@
   // alias of measurement, null pointer cannot be serialized in thrift so empty string is instead
   protected String measurementAlias = "";
 
+  private long majorVersion = 0;
+
+  private long minorVersion = 0;
+
```

Comment:
Add some comments to explain the two fields.

Reply:
done